doc/script_commands.txt

```diff
@@ -8174,6 +8174,15 @@ Opens the enchantgrade UI for the attached character or the player given by the
 
 This command requires packet version 2020-07-24 or newer.
 
+---------------------------------------
+
+*item_reform({<item id>{,<char id>}})
```

Comment:
How about allowing (or replacing the `<item id>` parameter with) the aegis item name since the design of `item_reform.yml` requires a name?
```suggestion
*item_reform({<item id>{,<char id>}})
*item_reform({<aegis item name>{,<char id>}})
```